### PR TITLE
Management

### PR DESCRIPTION
**Pull request by MistzSoftware**
This pull request includes several updates to the weekly reports and management artifacts in the documentation. The most important changes include adding detailed content for Week 7 in both the cumulative and individual weekly reports, and updating management artifacts with new links and a completed post-mortem section.

Updates to weekly reports:

* [`Documents/Management/WeeklyReports/cumulative.md`](diffhunk://#diff-a0017180b2348a9eb8abd8fa5784fc7bd54a7502e5bbf5941a5ab735c45e2fd5R421-R460): Added detailed sections for Week 7, including Overview, Accomplishments, Challenges, Plan for Next Week, Team Mood, and Conclusion.
* [`Documents/Management/WeeklyReports/week7.md`](diffhunk://#diff-d9450e13a2df79a8cd0a2745a45c32c58dff6b02dafda03d274a68062b6f3d9dR1-R41): Created a new file for Week 7 with comprehensive details on the week's activities, achievements, challenges, and future plans.

Updates to management artifacts:

* [`Documents/Management/managementArtifact.md`](diffhunk://#diff-b21ccb2a7a9148a56955a4e1131f85fa56f70b5baba801fae7706a4538d890f0R23): Added a link to the Gantt chart for better task visualization.
* [`Documents/Management/managementArtifact.md`](diffhunk://#diff-b21ccb2a7a9148a56955a4e1131f85fa56f70b5baba801fae7706a4538d890f0L86-L90): Completed the post-mortem section with detailed reflections from team members on their experiences, challenges, and learnings.

### CODE REVIEW

File: Documents/Management/WeeklyReports/week7.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   - Finalized the slideshow and practiced the presentation.  
+
+2. **Post-Mortem Review**  
+   - Conducted a project post-mortem to evaluate progress, highlight successes, and identify areas for improvement.  
+
+3. **Peer Evaluations**  
+   - Completed peer evaluations to provide feedback on individual contributions and foster better team collaboration.  
+
+4. **Implementing Level 2**  
+   - Integrated all sprites, sounds, and game mechanics to enhance the user experience for the oral presentation.
```
Comment (EnzoGuillouche) [inline Documents/Management/WeeklyReports/week7.md:19]: oral presentation does not exist in English, just write final presentation.

File: Documents/Management/WeeklyReports/week7.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Rating:** 7/10  
+
+The team experienced a noticeable reduction in stress levels, allowing for smoother collaboration and improved focus. However, instances of immature behavior were observed, which were not well-received and caused minor disruptions to the team dynamic. This highlights the importance of maintaining professionalism to foster a positive and productive environment.
+
+## Conclusion  
+
+This week marked a significant milestone with the completion of our presentation materials and the successful implementation of the second level. The team's dedication and ability to adapt under challenging circumstances, such as absences, showcased their commitment to the project. The post-mortem analysis and peer evaluations provided valuable insights and constructive feedback, setting a solid foundation for continuous improvement.
+
+Moving forward, the planned vacation will give the team a much-needed opportunity to recharge and return refreshed for the next phase of the project. Overall, the week was a success, with key accomplishments driving progress and areas of improvement identified for future collaboration.
```
Comment (EnzoGuillouche) [inline Documents/Management/WeeklyReports/week7.md:41]: "to recharge" would need precisions. such as "to recharge their batteries"

File: Documents/Management/managementArtifact.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ You can also find a detailed verison [HERE](projectCharter.md)
 # Tasks & schedules
 
 The breakdown of the tasks, their assignment, and schedule of those tasks can be found [HERE](https://sharing.clickup.com/9012521102/l/h/5-90123516211-1/1384ef34e36b04d).
+You can Also find the gantt chart [HERE](https://sharing.clickup.com/9012521102/g/h/8ck0a4e-412/37cdcab5cca6985)
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:23]: there's a capital letter at "also"

File: Documents/Management/managementArtifact.md [hunk 4]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Mathias<br>(SCRUM Master) | - Godot is very powerful<br> - Data about climat<br> - It's easier to keep a good workload when strict than when being a cool manager                                                                           | - Lack of project time to setup the project<br> - Huge lack of implication forcing me to be more a cop in a kinder garden than a real Project Manager                                                             | - Doing a lot of overtime<br> - Pausing few tasks to refocus desipated members | - Replace child by adults members<br> - Completing the game with more data and puzzles | - Working in cooperations with Enzo who was able to compare my current management with my management of an old project (Virtual processor team 2)<br> - The fact that few team members take on them to catch up lateness made by other members. | - Being a cops, all team members should be adults<br> - 3D generated environement | - The oral presentation                                                                                                          |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:89]: "desipated" should be "dissipated"

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:89]: replace **childish** members by **mature** ones

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:89]: The fact that few team members **took** on them to

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:89]: there should be no 's' for "Being a cops"

File: Documents/Management/managementArtifact.md [hunk 5]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Enzo                      | - First time project manager<br> - Learnt how to do a functional and use code<br> - Learnt what was the climat changes<br> - Unknown data about recycling and climat changes in general.                        | - Lack of project time to write the functional forced him to work during class and it wasn't good for learning<br> - catch up the time waste on the code, while learning how godot works and how it's structured. | - Doing a lot of overtime<br> - learn fast of to code in godot.                | - Serious game aspect, more puzzles and more data                                      | - Game design<br> - Being the program manager<br> - making a game                                                                                                                                                                               | - Tino's farts<br> - Lacking of implication<br> - lack of project time            | - The functional specifications<br> - About the advancement in the game developement in a so limited time |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:90]: First time PROGRAM manager

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:90]: there should be no 's' for "climat changes"

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:90]: catch up the time waste**D** on the code

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:90]: I believe you meant "learn fast **how** to code in godot."

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:90]: there no 'e' for "development"

File: Documents/Management/managementArtifact.md [hunk 6]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Evan                      | - Godot<br> - Detailed data about climat changes                                                                                                                                                                | - Lack of time to write the technical specification<br> - Decreasing of motivation due to lack of project<br> - time wasted due to sickness                                                                       | - Doing a lot of overtime<br> - Tried to remotivate<br> - heal the sickness    | - Serious game aspect, more puzzles and more data<br> - Better priorization of tasks   | - Game design<br> - game development<br> - good team                                                                                                                                                                                            | - Lack of project time<br> - technical specifications writing<br> - Tino's farts  | - To went so far even is obstacles encountered                                                            |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:91]: same about climate change

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:91]: "decrease of motivation" would be better

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:91]: 'heal from sickness' would be better

File: Documents/Management/managementArtifact.md [hunk 7]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Michel                    | - How does a game engine works<br> - How to well setup a game engine<br> - How to use godot<br> - How to write a good and structured code (thanks to the TL)<BR> - how it's difficult to create a serious game. | - Link files into another scene in godot<br> - underestimating a task's difficulty                                                                                                                                | - Persevering to solve issues<br> - Take time to understand the task           | - More features in the game like the lore, puzzles                                     | - Coding<br> - Work in a good team                                                                                                                                                                                                              | - Lack of project time<br> - Tino's farts                                         | - Code produced                                                                                           |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:92]: 'How does a game engine works' there should not be a s for work

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:92]: "how difficult it is" would be better

File: Documents/Management/managementArtifact.md [hunk 8]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Loic                      | - How to use godot<br> - Data about climat changes<br> - how to communicate with the team.<br> - What is a serious game.                                                                                        | - Merge with godot<br> - Crash on the windows<br> - Lack of project time.                                                                                                                                         | - Few overtime<br> - a bit of perseverance                                     | - More features in the game like the lore, puzzles                                     | - Good team<br> - Using godot<br> - the project in global<br> - level design                                                                                                                                                                    | - Tino's farts<br> - Lack of project time                                         | - Code produced<br> - Having a good team                                                                  |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:93]: same about climate change

File: Documents/Management/managementArtifact.md [hunk 9]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Tino                      | - What is a serious game<br> - Godot is shit<br> - The importance of reviewing multiple time a file.                                                                                                            | - Lack of indication about how to do our role                                                                                                                                                                     | - Asking to chatgpt, watching others repository                                | - More data and dialog in the game                                                     | - Good team ambiance<br> - musics                                                                                                                                                                                                               | - My own farts<br> - Being a QA (documentation)                                   | - My own farts<br> - Where the game is now<br> - good repartition at the oral presentation                |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:94]: same for final presentation

Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:94]: "multiple times" would be better

File: Documents/Management/managementArtifact.md [hunk 10]
```diff
@@ -83,11 +84,16 @@ The spreadsheet for the KPI's data and graphs can be found [HERE](https://docs.g
...
+| Axel                      | - I learnt Godot<br> - How to write a user manual<br> - Learnt about what is a serious game.                                                                                                                    | - Didn't knew how to do a user manual                                                                                                                                                                             | - finding a lot of resources on how to create it                               | - More lore and data                                                                   | - The team<br> - the user manual creation<br> - musics                                                                                                                                                                                          | - Tino's farts                                                                    | - The user manual<br> - Having a good team<br> - a good demo                                              |
```
Comment (EnzoGuillouche) [inline Documents/Management/managementArtifact.md:95]: "Didn't know" would be better

**Review by EnzoGuillouche — CHANGES_REQUESTED** [review]
Some issues have been spotted, and need to be solved before merging.

**Comment by MistzSoftware** [comment]
all of these issue have been fixed in commit https://github.com/algosup/2024-2025-project-2-serious-game-team-6/pull/42/commits/95f4dfcfef524abf54e15b054ff1fbe9523d5cc3

**Comment by MistzSoftware** [comment]
Forced merge because the QA deserted his post.

**Pull request merged**